cmd/proxy/Dockerfile

```diff
@@ -8,19 +8,14 @@ WORKDIR $GOPATH/src/github.com/gomods/athens
 
 COPY . .
 
-RUN apk update && \ 
-	apk add --no-cache bash && \
-	GO111MODULE=on CGO_ENABLED=0 go build -mod=vendor -o /bin/athens-proxy ./cmd/proxy && \
-	./scripts/create_default_config.sh
+RUN GO111MODULE=on CGO_ENABLED=0 go build -mod=vendor -o /bin/athens-proxy ./cmd/proxy
 
 FROM alpine
 
 ENV GO111MODULE=on
 
 COPY --from=builder /bin/athens-proxy /bin/athens-proxy
 COPY --from=builder /go/src/github.com/gomods/athens/config.dev.toml /config/config.toml
-COPY --from=builder /go/src/github.com/gomods/athens/cmd/proxy/locales /go/src/github.com/gomods/athens/cmd/proxy/locales
```

Comment:
could you please align handling of config? 
in olympus we do copy from workdir to /config in builder
and then copy from `/config/config.dev.toml` to `/config.toml`
while here we go directly to git repo dir

Reply:
@michalpristas this is one reason I'd like us to park Olympus. It's very outdated code at this point and wouldn't work anyway. I updated it tho :) .

Reply:
thanks, i would do it in a separate PR if you wouldnt though